app/assets/javascripts/item_selector/item_selector_zero_disable_submit.js

```diff
@@ -0,0 +1,48 @@
+var itemSelectorZeroDisableSubmit = (function() {
+
+  return $.extend(itemSelector, {
+    init: function() {
+      var _this = this;
+      $(document).on('ready page:load', function(){
+        if ($('[data-behavior="item-selector"]').length > 0) {
```

Comment:
After adding this check, I'm having a terrible time getting the teaspoon tests to pass (although I don't see why..) @jkeck, would you mind taking a look?